server/particleFunctions.js

```diff
@@ -6,6 +6,24 @@ const { helpers } = require('brave-alert-lib')
 
 const particle = new Particle()
 
+async function forceReset(deviceId, productId) {
```

Comment:
You could add a unit test for this by adding a file called `test/unit/particleFunctionsTest/forceResetTest.js`